provisioning/miniprov/hare_mp/store.py

```diff
@@ -99,18 +98,16 @@ def get_storage_set_index(self) -> int:
 
         raise RuntimeError('No storage set found. Is ConfStore data valid?')
 
-    def get_storage_set_nodes(self) -> List[str]:
-        storage_set_index = self.get_storage_set_index()
-
-        storage_nodes_key = (f'cluster>storage_set[{storage_set_index}]>nodes')
-        storage_nodes = self.get(storage_nodes_key)
-
-        for node in storage_nodes:
-            node_type = self.get(f'node>{node}>type')
-            # Skipping controller node
-            if node_type != 'storage_node':
-                storage_nodes.remove(node)
-
+    def get_data_nodes(self) -> List[str]:
```

Comment:
Let's have a single definition, there's one already present in provisioning/miniprov/hare_mp/utils.py - `get_io_nodes()` let's use that.

Reply:
**utils.py**
def get_io_nodes(self) -> List[str]:

		returning list of hostname
		
	Caller:
		`main.py` to get the list of hostname (server pods)
		
	output:
	    ['cortx-data-headless-svc-ssc-vm-g3-rhev4-2742']
		
**store.py**
def get_data_nodes(self) -> List[str]:

		returning list of machine_ID
		
	caller:
		`cdf.py` to get length or number of nodes (number of server pods)
		
		`test_cdf.py` will break if we remove, but we can replace it with `get_io_nodes()`
		
	output:
		['82efc1f2ccd64f378a68e9fc26797660']

Note: One of this function can be remove. I am in favor to remove  `get_io_nodes()` since it doesn't have much dependencies or effective area.